lib/modules/platform/azure/index.ts

```diff
@@ -527,6 +528,25 @@ export async function updatePr({
   } else if (state === 'closed') {
     objToUpdate.status = PullRequestStatus.Abandoned;
   }
+  if (platformOptions?.autoApprove) {
+    const pr: GitPullRequest = await azureApiGit.getPullRequest(
+      config.repoId,
+      prNo,
+      config.project
+    );
```

Comment:
```suggestion
    const pr = await azureApiGit.getPullRequestById(pullRequestId, config.project);
```

Reply:
I'm confused... pullRequestId doesn't exist here. Was your intention just to remove repoId?

Reply:
i replaced pullRequestId with prNo which I think is what you wanted to suggest

Reply:
yes, sorry, forgot to rename the variable after copy 🙃